res/rules-engine/configuration.toml

```diff
@@ -63,3 +63,20 @@ Type = "zero"
   Port = 5566
   Protocol = "tcp"
 
+# SecretStore is required when Store and Forward is enabled and running with security
+# so Database credentials can be pulled from Vault. Also now require when running with secure Consul
+# Note when running in docker from compose file set the following environment variables:
+#   - SecretStore_Host: edgex-vault
+[SecretStore]
+Type = 'vault'
+Host = 'localhost'
+Port = 8200
+Path = '/v1/secret/edgex/app-rules-engine/'
+Protocol = 'http'
+RootCaCertPath = ''
+ServerName = ''
+TokenFile = '/tmp/edgex/secrets/app-rules-engine/secrets-token.json'
```

Comment:
so we want `app-rules-engine` as the role name in security-bootstrapper side, right?

Reply:
Correct